Loop/Managers/Alerts/StoredAlert+CoreDataClass.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  StoredAlert+CoreDataClass.swift
+//  Loop
+//
+//  Created by Rick Pasetto on 5/22/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+//
+
+import Foundation
+import CoreData
+
+
+public class StoredAlert: NSManagedObject {
+
```

Comment:
If you want `modificationCounter` to automatically increment, you'll need this here:
```
    override func willSave() {
        if isInserted || isUpdated {
            setPrimitiveValue(managedObjectContext!.modificationCounter ?? 0, forKey: "modificationCounter")
        }
        super.willSave()
    }
```

Reply:
This is the same code as in the other model classes.

Reply:
@darinkrauss I'd put that code in `StoredAlert.swift` (an `extension`) -- it was there when I'd assumed `StoredAlert+CoreDataClass.swift` would be auto-generated.